openvox/install.md

```diff
@@ -79,6 +79,21 @@ apt::source { 'openvox8-release':
 }
 ```
 
+```puppet
+include yum
+$release=8
+$os_name = $facts['os']['name'] ? {
+  /(AlmaLinux)|(Rocky)|(RedHat)|(Centos)/ => "el",
```

Comment:
Maybe use default here instead, since 99% of the Redhat Derivates out there are el